src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
All magic numbers in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
```